Src/Support/Google.Apis.Auth.Tests/OAuth2/Responses/AuthorizationCodeResponseUrlTests.cs

```diff
@@ -68,6 +70,8 @@ private void SubtestConstructor(AuthorizationCodeResponseUrl response)
             Assert.Equal("URI", response.ErrorUri);
             Assert.Equal("DESC", response.ErrorDescription);
             Assert.Equal("STATE", response.State);
+            Assert.Contains("another_parameter", response.AdditionalParameters.Keys);
+            Assert.Equal("ANOTHER_PARAMETER", response.AdditionalParameters["another_parameter"]);
```

Comment:
To make sure we have only one.
```suggestion
            var pair = Assert.Single(response.AdditionalParameters);
            Assert.Equal("another_parameter", pair.Key);
            Assert.Equal("ANOTHER_PARAMETER", pair.Value);
```